apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java

```diff
@@ -163,6 +163,9 @@
    */
   static final String CODING_NPI_US = "http://hl7.org/fhir/sid/us-npi";
 
+  /** The Code System, as available at <a */
+  static final String CODING_CODESYSTEM = "http://terminology.hl7.org/CodeSystem/v2-0203";
```

Comment:
Could you please rename this to something less generic, e.g. `FHIR_IDENTIFIER_TYPE_CODESYSTEM`?